src/generated/dpms.rs

```diff
@@ -211,14 +214,14 @@ where
         suspend_timeout_bytes[1],
         off_timeout_bytes[0],
         off_timeout_bytes[1],
-        0 /* trailing padding */,
-        0,
     ];
     let length_so_far = length_so_far + request0.len();
+    let padding = [0; 2];
+    let length_so_far = length_so_far + padding.len();
```

Comment:
Can you try generating the padding in the last array instead of appending it as an extra array like this?
(I did a quick grep through the python generated code and it seems that this trailing padding is always after a fixed size part; `git grep -B 1 '0 /\* trailing padd`)

Reply:
Fixed